apps/src/lib/util/AnalyticsConstants.js

```diff
@@ -86,6 +86,8 @@ const EVENTS = {
   TA_RUBRIC_ON_STUDENT_WORK_UNLOADED: 'TA Rubric On Student Work Unloaded',
   TA_RUBRIC_SUBMITTED: 'TA Rubric Submitted',
   TA_RUBRIC_EVIDENCE_LEVEL_SELECTED: 'TA Rubric Evidence Level Selected',
+  TA_RUBRIC_RUN_BUTTON_CLICKED:
+    'TA Rubric - Teacher clicked RUN button on student work',
```

Comment:
tiny tiny nit: remove the dash to keep it consistent with the other event names?